### DISCUSSION TITLE
IP and standards body for the spec

### DISCUSSION THREAD

**Discussion by bmann** [ucan-wg/spec#31] [General]
@expede has had discussions with a number of people asking about what the plan is for IP rights and standardization.

We're going to talk to the [Decentralized Identity Foundation (DIF)](https://identity.foundation/). We're already members there and have participated in some discussions.

Our other path would be to do the simplest thing possible in this space to clear IP related to the spec.

There might be a future path to UCAN at the IETF, which is the other standards body that we think is a good fit.

Or, option 3? Please let us know what your thoughts and concerns are. We'll likely organize a live call with anyone interested in this topic to talk through what the members of this working group want.

Our main goal is to keep working with organizations that want to work on an implementable, interoperable spec.

---

UPDATE https://github.com/ucan-wg/spec/discussions/31#discussioncomment-2291771

**Comment by expede** [comment]
cc @b5 @dholms @mikeal @blaine as potential folks with stake and/or thoughts

**Comment by expede** [comment]
Also part of this discussion, we were forwarded this a few weeks back https://github.com/CommunitySpecification/1.0

**Comment by expede** [comment]
We got direct feedback _pretty quickly_ from several people after posting this 😅

The TL;DR is that managing the patent covenant and CLA hygiene stuff is essential. Housing in an org is not _impossible_ to reverse or change with enough buy in, but it is the least reversible choice. The spec if pretty early today, and it probably needs to stay nimble and rapidly iterable before e.g. moving it to the IETF.

We're going to set up CLAs and patent covenants, and push decision on an org to later, or until there's a forcing function.

This has been recommended by some folks at the DIF as one that would be compatible for them if we were to house UCAN with them in the future https://github.com/CommunitySpecification/1.0
